### PR TITLE
Tkrs updates

### PR DESCRIPTION
**Pull request by tkrstajic**

### CODE REVIEW

File: articles/ams-team-setups.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+In happy circumstances, the future members of the Operations team are involved first in the bid process (during Discovery phase) – like the Service Manager, the DevOps engineer as early as the Alpha phase (when the environments and CI/CD** get set up). Latest, these team members should get involved by the mid of Beta phase (or in other words Development or Delivery phase), in order to set up proper tools and processes to support Release Management, preventive maintenance as well as Incident Management.
+As the application approaches the Live phase, the team undergoes a shift in focus and setup. A Service Manager and a DevOps engineer get added to the team, and Developer and Quality Manager undergo a slight change of the mindset – their primary focus isn’t only delivering new features but also ensuring that the existing ones are performing in production and satisfying user needs.
```
Comment (peitor) [inline articles/ams-team-setups.md]: Is this description a good way of doing it? 
Or do you follow the pattern: Situation, Pain, Why, BetterWay.

To me the situation described sounds not very DevOpsy. 
Adding and removing people is not a good practice in my experience and not what DevOps means....

File: articles/ams-team-setups.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+In our projects we use three Live team*** setups and cover a scenario specific to the agile development frameworks – the delivery of the new features goes in parallel to the operations in production. Also, as the change always brings challenges, and such change in our case impacts mostly the Developer role, our scenarios are named after him.
```
Comment (peitor) [inline articles/ams-team-setups.md]: Maybe introduce this chapter with 
"Good Practices From Our Projects".
That makes it clear that this is not general DevOps best practice.

File: articles/ams-team-setups.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+authorName: Tijana Krstajic, Guido Angenendt
+authorGithubUsername: doogu
+issue: 91
+title: Operations Team Setups
```
Comment (peitor) [inline articles/ams-team-setups.md]: Can we make the title more Zühlke Focused?

Or the article less Zühlke focused: AMS, ADM,...
Inspiration for DevOps antipatterns can be found on https://web.devopstopologies.com/

File: articles/ams-team-setups.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+**FIXED DEVELOPER**
+
+![Dummy Placeholder Image](./images/ams/fixed.png)
+ 
+The first scenario we considered is ‘Fixed Developer’, a situation where there is a clear distinction between the delivery and Live teams and the Live team has a fixed developer assigned that deals only with Operations topics (Event Management, Incident Handling, performance enhancements etc.). The benefit of such an approach is that there is no capacity impact to the development, and there is a dedicated person to respond to the operations demands. The main con is the knowledge transfer from Delivery to Fixed Developer that needs to follow each new feature implemented into production. An additional downside is no full allocation to the Live team (in most cases it works like this) – the team cannot assure that the SLAs are always followed, and the constant project and context switching by the developer can cause a knowledge disperse. Considered the above, we came to the conclusion that this scenario is fit only for such setups where there is no more ongoing development.
```
Comment (peitor) [inline articles/ams-team-setups.md]: Is this Anti Type B?
https://web.devopstopologies.com/#anti-type-b

File: articles/ams-team-setups.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
...rarily (for a week or some weeks) join the Live team to be in charge of the operations topics that require development. The benefits of this approach are multiple – the knowledge transfer is rather lean, as each rotation brings knowledge from operations to Delivery and vice versa, and short rotations enable speedy knowledge spreading within the project. The developer stays fully allocated to the project, and his capacity is shared between Delivery and AMS. A great pro of this approach is definitely the ‘eat your own dog food’ moment, as the developers have a chance to maintain the code they have written. This effect often leads to better software and better software quality. The downside of the ‘RD’ scenario is that the developers cannot be fully assigned to backlog work. An additional point to consider is definitely the knowledge needs – the developer in the RD role needs to have a broad understanding of the system and technologies involved for the application. These are the main reasons we would not recommend this approach for applications with a wide spectrum of technologies (.Net, Mobile etc.). Also, this approach is good for customers who are strict in terms of delivery dates.
```
Comment (peitor) [inline articles/ams-team-setups.md]: Cool. I liked the description of the "rotating developer"

File: articles/ams-team-setups.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+The three Live team setups are a product of our learnings and it represents an evolution path – starting from Fixed Developer as a logical and simplest setup and arriving to One Team as a logical conclusion – the project team is one, as the overall goal should be one. We are a part of the project to ensure the success of our Customer in all areas that are in our responsibility.
```
Comment (peitor) [inline articles/ams-team-setups.md]: Aha! This is more an evolution. Cool. I didn't think of it like that.

Maybe the title of the article could reflect that?
Ideas:
* "DevOps - An Evolutionary Tale"  
* "The Evolution Of Our Operations Team Setup"
* "Things That Change When Transitioning Operation Teams"
* "Something Breaks When You Go DevOps"
* "Operation Teams - The Effects Of Collaboration"

File: articles/ams-team-setups.md (hunk 3 above)
Comment (tkrstajic) [reply articles/ams-team-setups.md]: AMS is a generally used term, not Zuehlke centric. ADM potentially we could eliminate, but that would be negating the Zuehlke project best practices...

**Pull request merged**